android-components/components/support/webextensions/src/main/java/mozilla/components/support/webextensions/ExtensionProcessDisabledPopupObserver.kt

```diff
@@ -27,9 +27,9 @@ open class ExtensionProcessDisabledPopupObserver(
 
     override fun start() {
         popupScope = store.flowScoped { flow ->
-            flow.distinctUntilChangedBy { it.showExtensionProcessDisabledPopup }
+            flow.distinctUntilChangedBy { it.showExtensionsProcessDisabledPrompt }
```

Comment:
This property in the store was named to match other popup features. I'm OK with the change but I think we want to update the class, lambda, and coroutinescope name here to match.